src/Core/src/Platform/iOS/ImageViewExtensions.cs

```diff
@@ -21,24 +22,41 @@ public static void UpdateAspect(this UIImageView imageView, IImage image)
 			imageView.ClipsToBounds = imageView.ContentMode == UIViewContentMode.ScaleAspectFill || imageView.ContentMode == UIViewContentMode.Center;
 		}
 
-		public static void UpdateIsAnimationPlaying(this UIImageView imageView, IImageSourcePart image)
+		public static async Task UpdateIsAnimationPlaying(this UIImageView imageView, IImageSourcePart image)
 		{
-			if (image.IsAnimationPlaying)
+			if (imageView is MauiImageView mauiImageView)
 			{
-				if (!imageView.IsAnimating)
-					imageView.StartAnimating();
-			}
-			else
-			{
-				if (imageView.IsAnimating)
-					imageView.StopAnimating();
+				// IsAnimationPlaying set to true in the incoming imagesource
+				// indicates it's an animated image.
+				if (image.IsAnimationPlaying)
+				{
+					if (mauiImageView.IsAnimating)
+					{
+						mauiImageView.StopAnimating();
+					}
+					
+					if (!imageView.IsAnimating)
+					{
+						var animatedImage = await ImageAnimationHelper.CreateAnimationFromImageSource(image.Source);
+						mauiImageView.AnimationImages = animatedImage.GetValuesAs<UIImage>();
+						mauiImageView.AnimationDuration = animatedImage.Duration;
+						mauiImageView.StartAnimating();
+					}
+				}
+				else
+				{
+					if (imageView.IsAnimating)
+					{
+						mauiImageView.StopAnimating();
+					}
+				}
 			}
 		}
 
 		public static void UpdateSource(this UIImageView imageView, UIImage? uIImage, IImageSourcePart image)
 		{
 			imageView.Image = uIImage;
-			imageView.UpdateIsAnimationPlaying(image);
+			imageView.UpdateIsAnimationPlaying(image).FireAndForget();
```

Comment:
This already uses fire and forget, so nothing weird is happening.

Reply:
If I remove that CS4014 is triggered which we consider an error. So I either need to add an `await` or use a discard: `_ = imageView.UpdateIsAnimationPlaying(image);`

Does it then matter which of those options we do either `FireAndForget()` or the discard?